### PR TITLE
opening WhatsNew page in internal browser

### PR DESCRIPTION
**Pull request by nnngrach**
https://github.com/osmandapp/OsmAnd-iOS/issues/897

### CODE REVIEW

File: Sources/Controllers/2.0/BottomSheet/OABaseBottomSheetViewController.h [hunk 1]
```diff
@@ -22,6 +22,7 @@
 
 @property (nonatomic, readonly) CGFloat initialHeight;
 
+- (CGFloat) getHeaderViewHeight;
```
Comment (PaulStets) [inline Sources/Controllers/2.0/BottomSheet/OABaseBottomSheetViewController.h:25]: Seems redundant. If you want to adjust the initial height of the BottomSheet, simply override the getInitialHeight method in the subclass. Also, note that the ```self.headerView.frame``` is aavailable in the subclass as well.

File: Sources/Controllers/2.0/BottomSheet/WhatsNewBottomSheet/OAWhatsNewBottomSheetViewController.m [hunk 2]
```diff
@@ -44,68 +47,59 @@ - (void) applyLocalization
 - (void) generateData
 {
     NSMutableArray *data = [NSMutableArray new];
-    
-    NSString *fullAppVersion = [[[NSBundle mainBundle] infoDictionary] objectForKey:@"CFBundleShortVersionString"];
```
Comment (PaulStets) [inline Sources/Controllers/2.0/BottomSheet/WhatsNewBottomSheet/OAWhatsNewBottomSheetViewController.m:48]: Why did you put this into the cellForRow method? It should be calculated once here.

File: Sources/Controllers/2.0/BottomSheet/WhatsNewBottomSheet/OAWhatsNewBottomSheetViewController.m [hunk 3]
```diff
@@ -44,68 +47,59 @@ - (void) applyLocalization
...
+    UIFont *semiboldFont = [UIFont systemFontOfSize:17 weight:UIFontWeightSemibold];
+    [attributedString addAttribute:NSFontAttributeName value:regularFont range:fullRange];
+    [attributedString addAttribute:NSFontAttributeName value:semiboldFont range:boldRange];
+    
+    return attributedString;
+}
+
+- (CGFloat) getViewHeight
+{
+    CGFloat width = DeviceScreenWidth - 2 * kHorisontelMargin;
```
Comment (PaulStets) [inline Sources/Controllers/2.0/BottomSheet/WhatsNewBottomSheet/OAWhatsNewBottomSheetViewController.m:84]: Again, this is incorrect, to change the inital anchor point, override the getInitialHeight method. Overriding getViewHeight will break scrolling.

File: Sources/Controllers/2.0/BottomSheet/WhatsNewBottomSheet/OAWhatsNewBottomSheetViewController.m (hunk 2 above)
Comment (nnngrach) [reply Sources/Controllers/2.0/BottomSheet/WhatsNewBottomSheet/OAWhatsNewBottomSheetViewController.m:48]: done

File: Sources/Controllers/2.0/BottomSheet/WhatsNewBottomSheet/OAWhatsNewBottomSheetViewController.m (hunk 3 above)
Comment (nnngrach) [reply Sources/Controllers/2.0/BottomSheet/WhatsNewBottomSheet/OAWhatsNewBottomSheetViewController.m:86]: done

File: Sources/Controllers/2.0/BottomSheet/WhatsNewBottomSheet/OAWhatsNewBottomSheetViewController.m [hunk 4]
```diff
@@ -44,68 +47,59 @@ - (void) applyLocalization
...
+    UIFont *semiboldFont = [UIFont systemFontOfSize:17 weight:UIFontWeightSemibold];
+    [attributedString addAttribute:NSFontAttributeName value:regularFont range:fullRange];
+    [attributedString addAttribute:NSFontAttributeName value:semiboldFont range:boldRange];
+    
+    return attributedString;
+}
+
+- (CGFloat)initialHeight
+{
+    CGFloat width = DeviceScreenWidth - 2 * kHorisontelMargin;
```
Comment (PaulStets) [inline Sources/Controllers/2.0/BottomSheet/WhatsNewBottomSheet/OAWhatsNewBottomSheetViewController.m:84]: kHorizontalMargin

**Pull request merged**